sig-security-tooling/vulnerability-mgmt/README.md

```diff
@@ -47,8 +47,12 @@ and SIG Security's Tooling sub-project
    runtimes, container registries, Node OS
 3. **Resolving license violations**: Allowed third party license policy can be
    found [here](https://github.com/cncf/foundation/blob/master/allowed-third-party-license-policy.md#approved-licenses-for-allowlist)
+   4. **Fixing unfixable vulnerabilities**: The focus of this effort is to apply fixes to
+   vulnerabilities for which a patch is released by owners of the vulnerable dependency
+   or base image. This scoping decision is made to maximize utilization of maintainer
+   time and attention span on high impact work.
```

Comment:
```suggestion
4. **Fixing unfixable vulnerabilities**: The focus of this effort is to apply fixes to
vulnerabilities for which a patch is released by owners of the vulnerable dependency
or base image. This scoping decision is made to maximize utilization of maintainer
time and attention span on high impact work.
```

Seems like the indentation is off. This should fix it, I think.

Reply:
Oops, thanks!